ISSUE_TEMPLATE.md

```diff
@@ -12,6 +12,7 @@ Tell us what you think should happen.
 ### Actual behavior
 
 If possible, please create a small demo that demonstrates the issue.
+You can fork https://jsfiddle.net/marionettejs/adhv48ky/ for quick demo setup.  
```

Comment:
What do you think about adding small note, that in fiddle used Mn v3, Radio v2 etc and if you are using another version, please don't forget to update resources in fiddle